ui/pages/Address.tsx

```diff
@@ -98,7 +107,10 @@ const AddressPageContent = () => {
     undefined;
 
   const isLoading = addressQuery.isPlaceholderData || (config.features.userOps.isEnabled && userOpsAccountQuery.isPlaceholderData);
```

Comment:
Should the condition about UserOps be moved into `isTabsLoading` too?

Reply:
reasonable